cmd/broker/ingress/main.go

```diff
@@ -92,7 +99,19 @@ func main() {
 		Path:   "/",
 	}
 
-	ceClient, err := cloudevents.NewDefaultClient()
+	kc := kubernetes.NewForConfigOrDie(mgr.GetConfig())
+	configMapWatcher := configmap.NewInformedWatcher(kc, env.Namespace)
+
+	zipkinServiceName := fmt.Sprintf("%s-broker-ingress.%s", env.Broker, env.Namespace)
```

Comment:
I think I'd like this to be a function so it can be unit tested.

Reply:
Done. Even added a test :)